packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -90,11 +90,11 @@ class Layout extends React.Component {
             <div className={cx('main-container-inner')}>
               <div className={cx('nav-burgerbar')}>
                 <Nav.Burger handleClick={this.toggleNav} />
-                {logo && logo.mobileLogo &&
-                  <div className={cx('mobile-logo')}>
+                <div className={cx('mobile-logo')}>
```

Comment:
The solution is that always display this img holder no matter if mobileLogo is existing or not. Assign a fixed height (35px which is the max-height of img) to this img holder to keep the flow.

Reply:
NIT: isn't it more appropriate for the nav-burgerbar to have the height of 35px?